### PR TITLE
MS1 basic project README + build script

### LINKED ISSUE #71: Project README + build script

### PR DESCRIPTION
**Pull request by gruvw**
This PR is documentation related.

It adds a short description of the project and its core features along with a link to the Penpot and the architecture diagram image as required per the MS1 deliverable requirements.

**Note** - this PR also include a build script that will be used to generate the APK in a consistent way from the CI.

Fixes #71

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+- **Offline Mode**: Caches posts for offline access, allowing users to view nearby content without an internet connection.
+
+## User Interface Design
+
+Link to the [Penpot Project](https://design.penpot.app/#/view/93d0ad32-dfe5-8194-8004-0171998eeabe?page-id=76cd5706-e69f-8069-8004-1a1578d3f0c7&section=interactions&index=25&share-id=a9010c45-753d-8133-8004-2d801bdcf756&interactions-mode=show) (view mode only).
+
+## Project Architecture Diagram
+
+Here is the first architecture diagram of the application for MS1.
+This should give a quick reference/overview on the overall architecture and the relations between the different screens, data sources, and internal state of the application.
```
Comment (TheTexanCodeur) [inline README.md:24]:
```suggestion
This should give a quick reference/overview of the overall architecture and the relations between the different screens, data sources, and internal states of the application.
```

File: scripts/build [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+flutter build apk --release
+
+VERSION=$(grep 'version:' pubspec.yaml | awk '{print $2}')
+mv "build/app/outputs/flutter-apk/app-release.apk" "build/Proxima-v$VERSION.apk"
```
Comment (TheTexanCodeur) [inline scripts/build:8]: Good idea to move the .apk to a more accessible folder and add the version in the name!

**Review by TheTexanCodeur — APPROVED** [review]
Good job! The description of the project is clear and showcases the main features of our app.

**Review by camillelnne — APPROVED** [review]
LGTM :)

**Pull request merged**